tests/sentry/api/endpoints/test_project_details.py

```diff
@@ -251,120 +252,145 @@ class ProjectUpdateTestTokenAuthenticated(APITestCase):
     def setUp(self):
         super().setUp()
         self.project = self.create_project(platform="javascript")
+        self.team = self.project.teams.first()
         self.user = self.create_user("bar@example.com")
+        self.url = reverse(
+            "sentry-api-0-project-details",
+            kwargs={
+                "organization_slug": self.project.organization.slug,
+                "project_slug": self.project.slug,
+            },
+        )
+        self.data = {"platform": "rust"}
+
+    def create_bearer_token(self, user, scope_list: List[str]) -> str:
+        token = ApiToken.objects.create(user=user, scope_list=scope_list)
+        return f"Bearer {token.token}"
 
-    def test_member_can_read_project_details(self):
+    def test_member_can_update_limited_project_details(self):
```

Comment:
what is the difference between limited and regular project details?

Reply:
Org members can only updated the `isBookmarked` and `isSubscribed` fields